test/Api/CatalogApiTest.php

```diff
@@ -264,10 +54,10 @@ protected function deleteTestCatalog() {
      *
      */
     public function test_batchDeleteCatalogObjects() {
-      $coffeeId = $this->idMap[self::COFFEE_CLIENT_ID];
-      $smallCoffeeId = $this->idMap[self::SMALL_COFFEE_CLIENT_ID];
-      $largeCoffeeId = $this->idMap[self::LARGE_COFFEE_CLIENT_ID];
-      $smallTeaId = $this->idMap[self::SMALL_TEA_CLIENT_ID];
+      $coffeeId = $this->catalogFixtures->idMap[CatalogFixtures::COFFEE_CLIENT_ID];
```

Comment:
I opted to refactor a bit to: `$coffeeId = $this->catalogFixtures->getObjectId(CatalogFixtures::COFFEE_CLIENT_ID);`

This hides the `idMap` behind a method. I'm generally not a fan of magic getters since they make the code hard to understand for new readers, and hard to grep.